examples/differential_xsec_plot.ma5

```diff
@@ -0,0 +1,25 @@
+# differential cross-section plot
+# c.f. http://madanalysis.irmp.ucl.ac.be/wiki/FAQNormalMode
```

Comment:
The main motivation for adding this is so that the Docker image build in CI has something to test against, but it seems to be a simple example to add also.